.github/pull_request_template.md

```diff
@@ -1,6 +1,5 @@
 <!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
 <!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->
-<!-- Your pull request title must be a Conventional Commits style message (https://www.conventionalcommits.org/en/v1.0.0/) -->
 <!-- If your pull request is going to change a lot of things, open an issue for discussion before starting your work. -->
```

Comment:
I have a weird suggestion. What if we remove this line too?

Reasoning:
- The benefit of this guidance is 80-90% the contributor's, and only 10-20% maintainers. e.g. it takes a little time for us to review a PR based on a bad idea and say sorry but this is not really what we want.. but most of the time wasted is theirs
- The only time someone is likely to see this template is when they have already pushed a branch and clicked "Create Pull Request".. i.e. they've already done the work
- Could this message discourage people who would otherwise submit a useful PR?
- Can we cover it in `CONTRIBUTING.md` instead?

Reply:
I've removed this line:

```markdown
<!-- If your pull request is going to change a lot of things, open an issue for discussion before starting your work. -->
```